extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking.tsx

```diff
@@ -23,67 +22,92 @@ function PanelMeasurementTableTracking({
+          </Button>
+          <Button
+            size="sm"
+            variant="ghost"
+            className="pl-0.5"
+            onClick={event => {
+              event.preventDefault();
+              event.stopPropagation();
+
+              commandsManager.runCommand('clearMeasurements', {
+                measurementFilter,
+              });
+            }}
+          >
+            <Icons.Delete />
+            Delete All
+          </Button>
+        </div>
+      </div>
+    );
+  }
+
+  const EmptyComponent = () => (
+    <MeasurementTable title="Measurements">
+      <MeasurementTable.Body />
+    </MeasurementTable>
+  );
+
   return (
     <>
-      <StudySummaryFromMetadata StudyInstanceUID={trackedStudy} />
       <PanelMeasurement
         servicesManager={servicesManager}
         extensionManager={extensionManager}
         commandsManager={commandsManager}
         measurementFilter={measurementFilter}
-        customHeader={({ additionalFindings, measurements }) => {
-          const disabled = additionalFindings.length === 0 && measurements.length === 0;
-
-          if (disableEditing || disabled) {
-            return null;
-          }
-
-          return (
-            <div className="bg-background flex h-9 w-full items-center rounded pr-0.5">
-              <div className="flex space-x-1">
-                <Button
-                  size="sm"
-                  variant="ghost"
-                  className="pl-1.5"
-                  onClick={() => {
-                    commandsManager.runCommand('downloadCSVMeasurementsReport', {
-                      measurementFilter,
-                    });
-                  }}
-                >
-                  <Icons.Download className="h-5 w-5" />
-                  <span className="pl-1">CSV</span>
-                </Button>
-                <Button
-                  size="sm"
-                  variant="ghost"
-                  className="pl-0.5"
-                  onClick={() => {
-                    sendTrackedMeasurementsEvent('SAVE_REPORT', {
-                      viewportId: viewportGrid.activeViewportId,
-                      isBackupSave: true,
-                    });
-                  }}
-                >
-                  <Icons.Add />
-                  Create SR
-                </Button>
-                <Button
-                  size="sm"
-                  variant="ghost"
-                  className="pl-0.5"
-                  onClick={() => {
-                    commandsManager.runCommand('clearMeasurements', { measurementFilter });
-                  }}
-                >
-                  <Icons.Delete />
-                  Delete All
-                </Button>
+        emptyComponent={EmptyComponent}
+        componentProps={{
+          grouping: {
+            header: props => (
+              <div>
+                <StudySummaryFromMetadata {...props} />
+                <CustomMenu {...props} />
               </div>
```

Comment:
This seems to be the same as StudySummaryWithActions (currently named StudySummaryMenu).